incubator/command-generate/src/test/resources/io/aklivity/zilla/runtime/command/generate/internal/asyncapi/http/proxy/complete/zilla.yaml

```diff
@@ -43,7 +43,8 @@ bindings:
         method: GET
         params:
           path:
-            id: string
+            id:
+              model: string
```

Comment:
Do we still support the abbreviated syntax for convenience?

Reply:
no, this is not supported anymore. Should we continue supporting this syntax?

Reply:
As discussed, we will be supporting abbreviated syntax for convenience.